menas/ui/components/login/loginMain.view.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright 2018-2019 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<core:View id="loginMainView" xmlns:core="sap.ui.core" xmlns:mvc="sap.ui.core.mvc" xmlns="sap.m" xmlns:form="sap.ui.layout.form"
+		   controllerName="components.login.loginMain">
+	<Page title="Login" enableScrolling="true">
+		<content>
+			<VBox id="loginForm" fitContainer="true" justifyContent="Center" alignItems="Center">
+				<items>
+					<Input type="Text" id="username" placeholder="username" value="{login>/username}" submit="onLoginSubmit"/>
+					<Input type="Password" id="password" placeholder="password" value="{login>/password}" submit="onLoginSubmit"/>
+					<Button width="10rem" text="Login" type="Emphasized" id="loginSubmit"/>
```

Comment:
I know there is no predetermined sequence of attributes, but I would like if we could keep it same for the whole view. So `type`, `id`, `..rest..`. Just for readability.

Reply:
Fair point.